src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.test.js

```diff
@@ -142,6 +142,13 @@ describe('addBylineBlock', () => {
     expect(addBylineBlock(input)).toEqual(input);
   });
 
+  it('should return input if page type is not CSP', () => {
+    const input = deepClone(styInput);
```

Comment:
This test is an exact duplicate of the one above, shouldn't there be an additional test to check a byline is added for CSP pages?

Reply:
let me look into this now.

Reply:
@andrewscfc I have added a new test testing that a byline is also added for csp assets